request.py

```diff
@@ -2,7 +2,7 @@
 

```

Comment:
GPT summary of [d93929](https://github.com/SampatSuthar/ml-demo/blob/8d6f43151d61cd79b56f97f6f5206c12ecd101ab/request.py#d93929484d2010cad4b95adcc06dbcda780171a3) - [4660c9](https://github.com/SampatSuthar/ml-demo/blob/f4c31c4d6e1e13ad49b9c7c04ec99df1d76c930c/request.py#4660c9f3d16279af0a343bf905e066c09d6fc196):
* Changed the condition of an `if` statement from `if :` to `if r==0:`